src/fprime_bootstrap/__main__.py

```diff
@@ -45,13 +47,47 @@ def main():
         help="Version of F´ to checkout (default: latest release)",
     )
 
+    clone_parser = subparsers.add_parser(
+        "clone", help="Clone an existing remote F´ project"
+    )
+    clone_parser.add_argument(
+        dest="url",
+        type=str,
+        help="URL of the remote repository to clone",
+    )
+    clone_parser.add_argument(
+        "--path",
+        type=str,
+        help="Path to create the project in (default: current directory)",
+        default=os.getcwd(),
+    )
+    clone_parser.add_argument(
+        "--no-venv",
+        action="store_true",
+        help="Do not create a virtual environment in the project",
+        default=False,
+    )
+    clone_parser.add_argument(
```

Comment:
Should this not be inferred by the project?  That is, a well-formed project would specify this.

Reply:
Done!